dependencies/binauralDecode.m

```diff
@@ -18,9 +21,10 @@
 
     %% rotate horizontally?
     if (nargin > 8 && ~isempty(horRotAngleRad) && horRotAngleRad ~= 0)
-        in = rotateHOA_N3D(in, horRotAngleRad*180/pi, 0, 0);
+        % TODO: This function requires 
+        %       https://github.com/polarch/Higher-Order-Ambisonics.git
+        in = rotateHOA_N3D(in, rad2deg(horRotAngleRad), 0, 0);
```

Comment:
The function requires the Higher Order Ambisonics toolbox but it is not called in the current test script, so I would say we keep it as is.